pkg/kubelet/volumemanager/reconciler/reconstruct_new.go

```diff
@@ -139,7 +142,7 @@ func (rc *reconciler) updateStatesNew(reconstructedVolumes map[v1.UniqueVolumeNa
 				klog.ErrorS(err, "Could not find device mount path for volume", "volumeName", gvl.volumeName)
 				continue
 			}
-			err = rc.actualStateOfWorld.MarkDeviceAsUncertain(gvl.volumeName, gvl.devicePath, deviceMountPath, "")
+			err = rc.actualStateOfWorld.MarkDeviceAsUncertain(gvl.volumeName, gvl.devicePath, deviceMountPath, seLinuxMountContext)
 			if err != nil {
 				klog.ErrorS(err, "Could not mark device is uncertain to actual state of world", "volumeName", gvl.volumeName, "deviceMountPath", deviceMountPath)
 				continue
```

Comment:
I am trying to verify(and I forgot some details), but is staging (ie. `MountDevice`) and Setup create different json files or they create same files? Are we consistently saving selinux context in both places? Is that feature gated?

Reply:
They create different jsons and I think SELinux is only in the SetUp one.

Reply:
Correction: it was in MountDevice, I added it to both + both feature-gated.